From 0af47c76cc2a4ee7be083d9e678810d9c5944420 Mon Sep 17 00:00:00 2001
From: Marius B. Kotsbak <marius@kotsbak.com>
Date: Sat, 01 Sep 2012 22:14:08 +0000
Subject: gsm: change to proper LTE entry in the ETSI mode table.

---
diff --git a/src/mm-generic-gsm.c b/src/mm-generic-gsm.c
index 2da7397..690f344 100644
--- a/src/mm-generic-gsm.c
+++ b/src/mm-generic-gsm.c
@@ -4575,7 +4575,7 @@ static ModeEtsi modes_table[] = {
     { MM_MODEM_GSM_ACCESS_TECH_HSDPA,       4 },
     { MM_MODEM_GSM_ACCESS_TECH_HSUPA,       5 },
     { MM_MODEM_GSM_ACCESS_TECH_HSPA,        6 },
-    { MM_MODEM_GSM_ACCESS_TECH_HSPA,        7 },  /* E-UTRAN/LTE => HSPA for now */
+    { MM_MODEM_GSM_ACCESS_TECH_LTE,         7 },
     { MM_MODEM_GSM_ACCESS_TECH_UNKNOWN,    -1 },
 };
 
--
cgit v0.9.0.2-2-gbebe
